core/services/configuration_file_manager.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+# Class for configuring network_config and configured_labels files
+class ConfigurationFileManager
```

Comment:
All methods of this "class" work with the NetworkConfig class. Therefore it is clear that these methods must be simply put into the NetworkConfig class.